### DISCUSSION TITLE
Shady Actions from Project Owners

### DISCUSSION THREAD

**Discussion by Neipas09** [rustdesk/rustdesk#1162] [General]
We are a team from Singapore. Sorry, I will lock your discussion soon, no politics allowed here.

_Originally posted by @rustdesk in https://github.com/rustdesk/rustdesk/discussions/1159#discussioncomment-3312105_

**Comment by Neipas09** [comment]
The fact that you closed/locked/deleted my post speaks volumes about how untrustworthy YOU and this app is.

My original post had NOTHING to do with politics, and everything to do with SECURITY, SAFETY, and PRIVACY!

**Comment by rustdesk** [comment]
That is ok if no politics like this post. Our public servers are for demo only, very cheap servers (vultr instance $5/month), and the EU servers are donated by contributors, https://github.com/rustdesk/rustdesk/discussions/1026. We encourage users set up their own servers or build both client and server from source code, our public servers have very limited bandwidth.

**Comment by derStephan** [comment]
>and the EU servers are donated by contributors

yes. But in the issue #1026 you expected full admin access to the server. It is indeed a bit fishy that you do not want foreign eyes on the server.
